src/sentry/static/sentry/app/views/releaseArtifacts.jsx

```diff
@@ -66,6 +67,29 @@ const ReleaseArtifacts = React.createClass({
     });
   },
 
+  handleRemove(id) {
```

Comment:
I feel like we should throw up a `confirm()` before deleting. Ignore me if this was planned. :)